### ISSUE TITLE
Add General/FileExtension to MediaInfo report for Policy Tests

### ISSUE THREAD

**Issue by dericed** [MediaArea/MediaConch_SourceCode#459]

**Comment by JeromeMartinez** [comment]
I disagree: I would like to keep the XML as small as possible, only with stuff in relation to the file itself if there is no issue detected. e.g. 99% of AVI files have the .avi extension, no need to put ".avi" in all outputs.

and also a test on file extensions should be done where file extensions are registered, so in MediaInfoLib.
Would be OK for you if there is a test in MediaInfo and MediaConch only tests a XML presence "e.g. "FileExtension_Error" field with the list of valid extensions only if file extension is not in the list)?
